htlcswitch/link.go

```diff
@@ -1950,11 +1950,12 @@ func (l *channelLink) handleUpstreamMsg(msg lnwire.Message) {
 			// us to proceed with an unclean state.
 			//
 			// NOTE: We do not trigger a force close because this
-			// could resolve itself in case our db was just busy not
-			// accepting new transactions.
+			// could resolve itself in case our db was just busy
+			// not accepting new transactions.
 			l.fail(
 				LinkFailureError{
-					code: ErrInternalError,
+					code:          ErrInternalError,
```

Comment:
This is still going to send an error to the peer, which according to the spec the peer should then force close.

If we really want to avoid FC, we should send a warning instead.

Reply:
Yeah I think that makes sense. Will update it to be a warning, as in theory we _may_ be able to recover here after a reconnect.